packages/types/src/event.ts

```diff
@@ -29,6 +29,12 @@ export const EventWriteSchema = EventSchema.partial({
 
 export type EventWrite = z.infer<typeof EventWriteSchema>
 
+export const EventFull = EventSchema.extend({
+  eventCommittees: z.array(EventCommitteeSchema),
+})
+
+export type EventFull = z.infer<typeof EventFull>
```

Comment:
This is an important change. I'm not sure if it's the best approach, but it makes some sense. We need the EventSchema to match the database schema because it is used by the event repository. However, the UI needs a type that includes other relations. Feedback here would be nice

Reply:
Let's name it EventWithCommittees